src/pretix/static/pretixpresale/scss/_checkout.scss

```
@@ -102,6 +102,9 @@
     }
   }
 }
+.cross-selling .panel-body > section:first-child > h3 {
+  margin-top: 10px; font-size: 21px;
+}
```

Why does only the first section get a smaller category-headline? Looks a bit strange if I have multiple cross-selling categories. Also – this ist very detailed nit-picking – the `margin-top` adds to much space compared to other addons headline inside a `.panel-body`. Also the default 45px margin-top for h3 is way bigger than the space when having multiple addon-categories. To match the spacing of addons, it needs to be:

```suggestion
.cross-selling .panel-body h3 {
  font-size: 21px;
  line-height: inherit;
  margin-top: 20px;
}
.cross-selling .panel-body > *:first-child > h3:first-child {
  margin-top: 0;
}
```

Note the `line-height: inherit` which increases the line-height to the same amount as the `<legend>` elements for the addons have and thus creates exactly the same space at the top of the `.panel-body`.

Arguably the `margin-top: 20px` might be a bit small, but if we were to increase it to e.g. 30px, we would need to change `.addons fieldset` as well.